build-android/build.sh

```diff
@@ -5,7 +5,7 @@ set -e
 # Config
 
 export SCONS="scons -j${NUM_CORES} verbose=yes warnings=no progress=no"
-export OPTIONS="production=yes"
+export OPTIONS="production=yes store_release=yes"
```

Comment:
@akien-mga I'd updated this section and moved the `store_release` option to the actual scons command (see https://github.com/godotengine/godot-build-scripts/pull/78/files).

Having it here means we're only generating the release builds and not the debug builds.

Reply:
That means double the build time, which is already pretty high. Do we really need those debug builds?

Reply:
> That means double the build time, which is already pretty high. Do we really need those debug builds?

Your call; I don't think we need them but you wanted the option of generating the debug builds for folks that are using the build scripts without a configured release key.

Reply:
Alternatively, either adding a descriptive comment, or checking the env variables to set the value for the `store_release` argument should do the trick without doubling the build time.

Reply:
Yeah I'll check the env variables to set the value properly.